frontend/src/metabase/query_builder/components/view/ViewFooter/LeftViewFooterButtonGroup.tsx

```diff
@@ -97,10 +104,12 @@ export const LeftViewFooterButtonGroup = ({
                 }}
               />
             </Tooltip>
+          ) : (
+            <Loader size="xs" />
```

Comment:
I can't see the loader in the active button:

https://github.com/user-attachments/assets/eb8e027b-cf3d-4fd6-ba95-bfa89a6a4c7b

Reply:
should be fixed now

https://github.com/user-attachments/assets/9459a844-73e1-4260-835e-660bb8d25d7d